src/targets/gpu/include/migraphx/gpu/gemm.hpp

```diff
@@ -168,6 +169,7 @@ struct rocblas_gemm
                 solution_idx = gemmImpl.validate(ctx, input_shapes, solution_idx);
             }
         }
+#endif
```

Comment:
We also need to consider cases where MIGraphX wasn't compiled with rocBLAS BETA but MXR passed MiGraphX was created with BETA.  In that case, we can't use solution_index other than `0` since it wouldn't be supported by rocBLAS that was linked with MIGraphX during compile time.